src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -1,17 +1,36 @@
 package edu.kis.vh.nursery;
 
-public class HanoiRhymer extends defaultCountingOutRhymer {
+public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-    int totalRejected = 0;
+    private int totalRejected = 0;
 
+    //TODO: DELETE reportRejected, does the same thing as getTotalRejected

```

Comment:
encapsulation of this field is unnecessary. reportRejected is part of its API so getTotalRejected should be deleted